src/Repository/DoctrineConfig.php

```diff
@@ -32,11 +39,14 @@ class DoctrineConfig implements ConfigRepositoryInterface
      *
      * @param EntityManagerInterface $entityManager
      *   Doctrine EntityManager that can store and retrieve EntityConfigItem.
+     * @param bool|null $autoFlush
+     *   Set to false if you don't want the setter to flush the config value. Defaults to true.
      */
-    public function __construct(EntityManagerInterface $entityManager)
+    public function __construct(EntityManagerInterface $entityManager, ?bool $autoFlush = null)
```

Comment:
This should set the default in the method signature.

Reply:
fixed.